src/LoggedStatement.php

```diff
@@ -13,16 +13,23 @@
 use PDO;
 use PDOStatement;
 
+/**
+ * @phpstan-import-type  logEntry from Connection
+ */
 class LoggedStatement extends PDOStatement
 {
+    /**
+     * @param callable $queryLogger
+     * @param logEntry $logEntry
+     */
     protected function __construct(
         protected mixed /* callable */ $queryLogger,
```

Comment:
With the `@param callable` in the docblock, you can remove the mid-line comment here.